Newsletters/SecretDecks® Newsletter to Occultish.md

```diff
@@ -1,20 +1,22 @@
 ## Calling all creators! 
 SecretDecks® looks to you if you’ve ever reinterpreted the Tarot and made your own deck... or perhaps you feel inspired to do so? 
+
 ## Are you an artist?  
-Build a new global fanbase, make a living off your digital art without any middlemen and through the medium of your choice, as well as own and manage your copyrights!
-## What if you're not an aritst? 
+Build a new global fanbase and make a living off your digital art without any middlemen and through the medium of your choice, as well as own and manage your copyrights!
```

Comment:
looks good

Reply:
Ah yes. I used 'on" because it's sold on a platform, but "from" also works!